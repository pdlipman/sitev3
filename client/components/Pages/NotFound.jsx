import React from 'react';

import IconHeader from '../Layout/Header/IconHeader.jsx';

const NotFound = () => (
    <div>
        <IconHeader
            contents='Page Not Found'
            headerTag='h2'
            iconName='info'
            subHeader={'The page you were looking for cannot be found.'}
        />
    </div>
);

export default NotFound;
