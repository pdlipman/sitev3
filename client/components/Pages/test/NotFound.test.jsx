import React from 'react';
import { shallow } from 'enzyme';

import NotFound from '../NotFound.jsx';

describe('Pages/NotFound.jsx', () => {
    it('NotFound should render properly', () => {
        const wrapper = shallow(
            <NotFound>Hello Jest!</NotFound>,
        );
        expect(wrapper).toMatchSnapshot();
    });

});
