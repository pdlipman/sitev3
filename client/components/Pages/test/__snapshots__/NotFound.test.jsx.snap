// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages/NotFound.jsx NotFound should render properly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <NotFound>
    Hello Jest!
  </NotFound>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": null,
    "nodeType": "host",
    "props": Object {
      "children": <IconHeader
        contents="Page Not Found"
        headerTag="h2"
        iconName="info"
        subHeader="The page you were looking for cannot be found."
      />,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": null,
      "nodeType": "class",
      "props": Object {
        "contents": "Page Not Found",
        "headerTag": "h2",
        "iconName": "info",
        "subHeader": "The page you were looking for cannot be found.",
      },
      "ref": null,
      "rendered": null,
      "type": [Function],
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": null,
      "nodeType": "host",
      "props": Object {
        "children": <IconHeader
          contents="Page Not Found"
          headerTag="h2"
          iconName="info"
          subHeader="The page you were looking for cannot be found."
        />,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": null,
        "nodeType": "class",
        "props": Object {
          "contents": "Page Not Found",
          "headerTag": "h2",
          "iconName": "info",
          "subHeader": "The page you were looking for cannot be found.",
        },
        "ref": null,
        "rendered": null,
        "type": [Function],
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
